mne/io/kit/constants.py

```diff
@@ -13,101 +14,127 @@
 KIT.SHORT = 2
 KIT.INT = 4
 KIT.DOUBLE = 8
-KIT.STRING = 128
 
-# pointer locations
-KIT.AMPLIFIER_INFO = 112
```

Comment:
are the constants being replaced with just their values? if so, I don't see them all being subbed out.

Reply:
Yeah since most of the pointer locations were used just once I inserted them in the code directly, I thought it was easier to read (in particular because it's easier to compare with the c-code this is based on which does not pre-define offsets as variables).

Reply:
ah, sorry, github didn't load the changes in the `kit.py` because it said that there were too many changes, I see them now